helm/hpcc/templates/_helpers.tpl

```diff
@@ -677,7 +677,7 @@ data:
 {{- $categories := list "system" -}}
 {{- if has "data" .me.access -}}
 {{- $categories := append $categories "storage" -}}
-{{- end -}}
+{{- end }}
```

Comment:
This is the actual fix, the rest of the changes are just moving early code inside an exception handler, so we get to catch/log the problem.